reqs/tests/import_reqs_tests.py

```diff
@@ -81,7 +81,7 @@ def test_imports_correctly2(csv_file2):
     call_command('import_reqs', str(csv_file2))
 
     reqs = list(Requirement.objects.order_by('req_id'))
-    assert len(reqs) == 2
+    assert len(reqs) == 3
```

Comment:
I don't think we need to add that in this test; this was a hack around all three rows now getting imported since we're now not rejecting duplicates.